src/KubernetesClient/KubernetesClientConfiguration.ConfigFile.cs

```diff
@@ -560,6 +566,49 @@ public static K8SConfiguration LoadKubeConfig(Stream kubeconfigStream)
             return LoadKubeConfigAsync(kubeconfigStream).GetAwaiter().GetResult();
         }
 
+        /// <summary>
+        ///    Loads Kube Config from KUBECONFIG environment variable
+        /// </summary>
+        /// <param name="useRelativePaths">When <see langword="true"/>, the paths in the kubeconfig file will be considered to be relative to the directory in which the kubeconfig
+        /// <returns>Instance of the <see cref="K8SConfiguration"/> class</returns>
+        public static K8SConfiguration LoadKubeConfigFromEnvironmentVariable(string environmentVariable = "KUBECONFIG", bool useRelativePaths = true)
+        {
+            return LoadKubeConfigFromEnvironmentVariableAsync().GetAwaiter().GetResult();
+        }
+
+        /// <summary>
+        ///    Loads Kube Config from KUBECONFIG environment variable
+        /// </summary>
+        /// <param name="useRelativePaths">When <see langword="true"/>, the paths in the kubeconfig file will be considered to be relative to the directory in which the kubeconfig
+        /// <returns>Instance of the <see cref="K8SConfiguration"/> class</returns>
+        public static async Task<K8SConfiguration> LoadKubeConfigFromEnvironmentVariableAsync(string environmentVariable = "KUBECONFIG", bool useRelativePaths = true)
+        {
+            var kubeconfig = Environment.GetEnvironmentVariable(environmentVariable);
+
+            if (kubeconfig == null)
+            {
+                throw new KubeConfigException($"Environment variable KUBECONFIG is empty.");
```

Comment:
Since the environment variable name is configurable, you should probably use `environmentVariable`'s value in the error message.